src/index.js

```diff
@@ -43,32 +43,35 @@ function parse(obj, classname, isInsideObj) {
     }
   }
   if (!isInsideObj) {
-    arr[0] = wrap(arr[0], classname)
+    arr[0] = wrap(arr[0], "")
   }
   return arr
 }
 
-export default function(h) {
-  return function(nodeName) {
-    var cache = {}
+export default function(h, retObj) {
```

Comment:
I strongly advise against the second parameter being a simple boolean and instead use an object. This would allow for further improvements/configuration in the future:

```js
const p = picostyle(h, {
  returnObject: true,
  prefix: 'my-lib'
})
```

Reply:
It's like you read my mind.  I was thinking about implementing both an options object parameter and prefix support, but wasn't sure if I wanted to introduce such a drastic change in one pull request.  I'll modify this pull request to support both.

I'm planning to implement the prefix as:
```javascript
your-prefix#
```

Reply:
Great! Though I think it's better to have a separate PR for the prefix - one PR per feature :)